src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -245,8 +204,13 @@ def setup(self, builder):
         )
 
     def on_initialize_simulants(self, pop_data):
+        initial_propensity = self.randomness.get_draw(pop_data.index).rename(
+            f"initial_{self.state_column}_propensity"
+        )
+        self.population_view.update(initial_propensity)
+
         population = self.population_view.subview(
-            ["age", "sex", "static_child_wasting_propensity"]
+            ["age", "sex", "initial_child_wasting_propensity"]
```

Comment:
How does this work? Doesn't this component create the `initial_child_wasting_propensity` column? This should error when initializing the population, as this column won't exists yet.